### PR TITLE
Add VM Prerequisites to Getting Started Guide

### LINKED ISSUE #266: Update Getting Started Guide for Ubuntu VM pre-requisites
When using Ubuntu VM as BYO Hosts, there are certain pre-requisites that need to be met. The guide for this is distributed in different docs. We need to add them all together in one place in the [Getting Started Guide](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/docs/getting_started.md).
VM pre-requisites:
- manually install these packages - https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/docs/local_dev.md#pre-requisites-1
- update `/etc/hosts` with the output of `hostname` - sometimes we have seen `kubelet` error of host not found / reachable

### PR DESCRIPTION
**Pull request by anusha94**
**What this PR does / why we need it**:
- add packages installation command
- update k8s version to v1.22.3

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #266

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -97,8 +97,31 @@ done
 ### Register BYOH host to management cluster
 
 ---
+#### VM Prerequisites
+- The following packages must be pre-installed on the VMs
+  - socat
+  - ebtables
+  - ethtool
+  - conntract
```
Comment (dharmjit) [inline docs/getting_started.md:105]:
```suggestion
  - conntrack
```

**Review by dharmjit — COMMENTED** [review]
LGTM except for a typo.

File: docs/getting_started.md [hunk 2]
```diff
@@ -150,7 +173,7 @@ Running the following command(on the host where you execute `clusterctl` in prev
 ```shell
 $ CONTROL_PLANE_ENDPOINT_IP=10.10.10.10 clusterctl generate cluster byoh-cluster \
     --infrastructure byoh \
-    --kubernetes-version v1.22.0 \
+    --kubernetes-version v1.22.3 \
```
Comment (shamsher31) [inline docs/getting_started.md:176]: Do you also want to update the base image in the makefile to have a consistent experience?
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/4d149bdfb599cf5e434418e25b2e4b8491d31996/Makefile#L12

Comment (anusha94) [reply docs/getting_started.md:176]: You are right. I have opened a new issue https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/268 to track this + some other info about `skip-installation` flag that needs to be used when using docker hosts.

**Review by dharmjit — APPROVED** [review]

**Review by shamsher31 — APPROVED** [review]

**Pull request merged**